InnerLoopAreas.props

```diff
@@ -7,5 +7,6 @@
     
     <!-- Note that after modifying the file, you will need to close and reopen the solution -->
     <!-- for changes to be picked up correctly by Visual Studio and MSBUILD -->
+    <FeatureExpanderEnabled>true</FeatureExpanderEnabled>
```

Comment:
Should this change be omitted, as it's a local-only configuration?

Reply:
Yes changes to InnerloopArea.props file should not be checked in.